applications/audio/broadcast_configuration_tool/src/main.c

```diff
@@ -293,37 +272,35 @@ static void audio_send(void const *const data, size_t size, uint8_t num_ch)
 	prev_ret = ret;
 }
 
-static void broadcast_create(struct broadcast_source_big *broadcast_param,
-			     struct bt_bap_lc3_preset *preset)
+static void broadcast_create(uint8_t big_index)
 {
-	static enum bt_audio_location location[2] = {BT_AUDIO_LOCATION_FRONT_LEFT,
-						     BT_AUDIO_LOCATION_FRONT_RIGHT};
-	static struct subgroup_config subgroups[2];
-
-	for (size_t i = 0; i < ARRAY_SIZE(subgroups); i++) {
-		subgroups[i].group_lc3_preset = *preset;
-		subgroups[i].num_bises = 2;
-		subgroups[i].context = BT_AUDIO_CONTEXT_TYPE_MEDIA;
-		subgroups[i].location = location;
+	if (big_index >= CONFIG_BT_ISO_MAX_BIG) {
+		LOG_ERR("BIG index out of range");
+		return;
 	}
 
-	broadcast_param->packing = BT_ISO_PACKING_INTERLEAVED;
+	struct broadcast_source_big *brdcst_param = &broadcast_param[big_index];
 
-	broadcast_param->encryption = false;
+	for (size_t i = 0; i < ARRAY_SIZE(subgroups[big_index]); i++) {
+		if (subgroups[big_index][i].num_bises == 0) {
+			/* Default to 2 BISes */
+			subgroups[big_index][i].num_bises = 2;
+		}
 
-	bt_audio_codec_cfg_meta_set_bcast_audio_immediate_rend_flag(
-		&subgroups[0].group_lc3_preset.codec_cfg);
+		if (subgroups[big_index][i].context == 0) {
+			subgroups[big_index][i].context = BT_AUDIO_CONTEXT_TYPE_MEDIA;
```

Comment:
Suggest a WRN print if this is not set explicitly.

Reply:
Discuss! I reckoned that some variables could have default values and then those could be changed if needed/wanted